web/plaid.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
+  <meta name="viewport" content="width=device-width">
+  <meta charset="utf-8" />
+
+  <!-- Ensure the file is being opened inside an iFrame parent -->
+  <script type="text/javascript">
+    if (window.self === window.top) {
+      window.stop();
+    }
+  </script>
+  <script src="https://cdn.plaid.com/link/v2/stable/link-initialize.js"></script>
+  <title>Plaid</title>
```

Comment:
I think this can be removed?

Reply:
It's an HTML page, so i guess we can leave it.

Reply:
I think since it's not displayed or used, it should be removed.